acapy_wallet_upgrade/strategies.py

```diff
@@ -526,6 +530,16 @@ async def get_wallet_info(self, uri: str):
                 settings["wallet.key"],
             )
 
+    async def check_for_leftover_wallets(self, old_conn, migrated_wallets):
+        retrieved_wallets = await self.retrieve_wallet_ids(old_conn)
+        leftover_wallets = [
+            wallet for wallet in retrieved_wallets if wallet not in migrated_wallets
+        ]
+        if len(leftover_wallets) > 0:
+            LOGGER.warning(
+                f"The following wallets were not migrated: {leftover_wallets}"
+            )
```

Comment:
This is the right idea but since we've not used or setup a logger elsewhere, I think we'll need to print this warning in order to make sure it's actually visible to the user